wandb/sdk/launch/runner/gcp_vertex.py

```diff
@@ -0,0 +1,244 @@
+import datetime
+import os
+import subprocess
+import time
+from typing import Any, Dict, List, Optional
+
+from google.cloud import aiplatform  # type: ignore
```

Comment:
We should only import aiplatform within the functions that use it. Using the utility `get_module` function and indicating to install it if it fails.